packages/runtime/runtime-utils/src/summarizerNode/summarizerNodeWithGc.ts

```diff
@@ -441,16 +451,10 @@ class SummarizerNodeWithGC extends SummarizerNode implements IRootSummarizerNode
 		 * snapshot and the child node wasn't created then. If a child is created after that, its GC details should be
 		 * the one from the downloaded snapshot and not the base GC details.
 		 */
-		const getChildBaseGCDetailsP = new LazyPromise<IGarbageCollectionDetailsBase>(async () => {
-			// Ensure that the base GC details is loaded because a child can be created before GC runs which is when
-			// base GC details is usually loaded.
-			await this.loadBaseGCDetails();
-			const childBaseGCDetails = unpackChildNodesGCDetails({
-				gcData: this.gcData,
-				usedRoutes: this.usedRoutes,
-			});
-			return childBaseGCDetails.get(id) ?? {};
-		});
+		const getChildBaseGCDetailsFn = async () => {
+			const childNodesBaseGCDetails = await this.childNodesBaseGCDetailsP;
+			return childNodesBaseGCDetails.get(id) ?? {};
+		};
 
```

Comment:
How bad is the original bugs ? AB#2962 and AB#2917 ?

Reply:
2962 is the one that is the problem. It should not happen very often but if it does, it can be bad because the GC data is incorrect. It only affects GC though.